### PR TITLE
[HYD-256] update metabase guide

### PR DESCRIPTION
**Pull request by ljcesca**

### CODE REVIEW

File: analyze/with-metabase.md [hunk 1]
```diff
@@ -8,81 +8,31 @@ As [Metabase](https://www.metabase.com/) puts it:
...
-  * Wait for the Hydra instance to complete first-time provisioning by looking for the log line `initialized a new cluster`
-*   In another terminal let’s create a sample database and seed some data. You will be prompted for the Hydra password twice. It is (as configured in compose) `hydra`.
+* Create a sample database and seed some data. You will be prompted for the Hydra password twice. It is (as configured in compose) `hydra`.
 
     ```
-    createdb -h localhost -p 6432 -U postgres sample_data
-    psql -h localhost -p 6432 -U postgres -d sample_data \
-        -c "CREATE TABLE data (id uuid NOT NULL PRIMARY KEY DEFAULT gen_random_uuid(), sample integer, timestamp timestamptz) USING columnar;" \
-      -c "INSERT INTO data (sample, timestamp) SELECT floor((random() + random() + random() + random() + random() + random()) / 6 * 100000000)::int, to_timestamp(EXTRACT(epoch from NOW()) - floor(random() * 2600000)) FROM generate_series(1, 50000);"
+    psql $PGCONN \
```
Comment (wuputah) [inline analyze/with-metabase.md]:
```suggestion
    psql "$PGCONN" \
```

File: analyze/with-metabase.md [hunk 2]
```diff
@@ -148,4 +98,8 @@ Now let’s combine those questions into a dashboard to give our users at a glan
 
 #### Cleanup
 
-Once you’re done exploring Metabase and Hydra you can stop docker compose by using `control-c` to shutdown the containers. To remove the containers use `docker compose down` and to remove the containers and cleanup Metabase’s metadata and the sample data in Hydra use `docker compose down -v`.
+Once you’re done exploring Metabase and Hydra you can cleanup the sample data be droping the `sample_data` table.
+
+```
+psql $PGCONN -c "DROP TABLE sample_data;"
```
Comment (wuputah) [inline analyze/with-metabase.md]:
```suggestion
psql "$PGCONN" -c "DROP TABLE sample_data;"
```

**Review by wuputah — APPROVED** [review]

File: analyze/with-metabase.md (hunk 1 above)
Comment (wuputah) [reply analyze/with-metabase.md]: you can skip these if you've confirmed it works without quotes, I _think_ I've run into issues not using quotes before but not 100% sure.

Comment (ljcesca) [reply analyze/with-metabase.md]: Its a good idea incase someone is using a weird shell.

File: analyze/with-metabase.md (hunk 2 above)
Comment (ljcesca) [reply analyze/with-metabase.md]: Resolved.

**Pull request merged**